github.js

```diff
@@ -338,6 +338,16 @@
         });
       };
 
+      // List all collaborators of a repository
+      // --------
+
+      this.listCollaborators = function(cb) {
+        _request('GET', repoPath + "/collaborators", null, function(err, collabs) {
```

Comment:
This line can be changed in:

`_request('GET', repoPath + "/collaborators", null, cb);`